components/Grid/grid.css

```diff
@@ -0,0 +1,19 @@
+.grid{
+    display : grid;
+}
+
+.grid-ratio-50-50{
+    grid-template-columns: repeat(2, 1fr);
+}
+
+.grid-ratio-30-70{
```

Comment:
technically you should have written 3fr 7fr for 30-70 grid. Other than that everything is great. Good work.